<?xml version="1.0"?>

<launch>
<!--
<arg name="fake_gripper" default="false"/>

<group if="$(arg fake_gripper)">
  <node pkg="manipulation_fake_tool"  type="manipulation_fake_tool_server" name="manipulation_fake_tool_pick" args="/robotiq_gripper" output="screen"/>
</group>

<group unless="$(arg fake_gripper)">
  <node pkg="robotiq_ur_modbus"  type="test_robotiq_ros.py" name="robotiq_ur_modbus_pick" args="/robotiq_gripper" output="screen"/>
</group>
-->
<arg name="pipeline" default="dirrt" />
<arg name="load_param" default="true" />
<arg name="debug" default="false" />
<arg name="baseline" default="false"/>
<rosparam if="$(arg load_param)" command="load" file="$(find hrc_mosaic_test)/config/ik.yaml"/>
<!--<include file="$(find hrc_mosaic_test)/launch/static_scene.launch"/> -->


<include file="$(find hrc_mosaic_test)/launch/pick.launch">
<arg name="debug" value="$(arg debug)"/>
<arg name="load_param" value="$(arg load_param)"/>
<arg name="pipeline" value="$(arg pipeline)"/>
<arg name="baseline" value="$(arg baseline)"/>
<arg name="objects_description_file" value="agents_synergy_test/objects_description"/>
<arg name="objects_distribution_file" value="agents_synergy_test/objects_distribution"/>
<arg name="objects_grasp_description_file" value="agents_synergy_test/objects_grasp_description"/>
<arg name="boxes_distribution_file" default="agents_synergy_test/boxes_distribution" />

</include>


<include file="$(find hrc_mosaic_test)/launch/place.launch">
<arg name="debug" value="$(arg debug)"/>
<arg name="load_param" value="$(arg load_param)"/>
<arg name="pipeline" value="$(arg pipeline)"/>/
<arg name="baseline" value="$(arg baseline)"/>
<arg name="slots_distribution_file" value="agents_synergy_test/slots_distribution"/>
</include>


<include file="$(find hrc_mosaic_test)/launch/goto.launch">
<arg name="debug" value="$(arg debug)"/>
<arg name="load_param" value="$(arg load_param)"/>
<arg name="pipeline" value="$(arg pipeline)"/>
<arg name="baseline" value="$(arg baseline)"/>
</include>


</launch>
